testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/infinispan/counter/InfinispanCounterTestCase.java

```diff
@@ -67,6 +70,11 @@ private static Archive<?> createDeployment() {
                 .addPackage(InfinispanCounterServlet.class.getPackage())
                 .setManifest(new StringAsset("Manifest-Version: 1.0\nDependencies: org.infinispan, org.infinispan.commons, org.infinispan.counter\n"))
                 .addAsWebInfResource(EmptyAsset.INSTANCE, "beans.xml")
+                .addAsManifestResource(createPermissionsXmlAsset(
+                        // WFLY-17968
+                        new FilePermission("<<ALL FILES>>", "read"),
```

Comment:
What is the need here for `<<ALL FILES>>`? If it's for a module there is a new [utility](https://github.com/wildfly/wildfly-arquillian/blob/71f4fdbe5edd2a881a148d868569aee6720caaf3/wildfly-testing-tools/src/main/java/org/wildfly/testing/tools/deployments/DeploymentDescriptors.java#L338-L360) you can use in WildFly Arquillian to add permissions to a specific module. This would look something like:
```java
.addAsManifestResource(
        DeploymentDescriptors.createPermissionsXmlAsset(
                DeploymentDescriptors.addModuleFilePermission("org.eclipse.yasson"),
                new RuntimePermission("getClassLoader")),
        "permissions.xml")
```

Reply:
Thanks @jamezp - good to know!